src/website/shared/apps.py

```diff
@@ -7,3 +8,10 @@ class SharedConfig(AppConfig):
 
     def ready(self) -> None:
         import shared.listeners  # noqa
+        from shared.auth import reset_group_permissions
+
+        # Configuration of signals
+
+        # Reset general group perimissions after migrations to
+        # include potential new permissions added.
+        post_migrate.connect(reset_group_permissions, sender=self)
```

Comment:
Can you provide a concrete usecase for this? I'm not exactly sure it makes sense.

Reply:
When new tables are created, new permission values get created. These will have to be assigned to admins, at least (superusers bypass permissions and get access to everything).

The alternative is getting rid of this signal and add the function as part of a relevant migration (or as a subsequent migration) when we notice some permissions being missed by admins.

The part of that function that relates to readers could be left apart as more  tables prefixed with nix won't probably be added.

Reply:
Permissions are now based on custom view mixins.